build_tools/requirements_conda.txt

```diff
@@ -7,3 +7,4 @@ nltk==3.2.1
 pytest==2.9.1       
 flask==0.12         # server
 requests==2.11.1
+matplotlib==1.5.3
```

Comment:
Oh, btw i've named the commit "adding matplotlib to requirements_pip" while in reality i was adding it into requirements_conda.

Ok, so i'm adding `pip install -r ./build_tools/requirements_pip.txt` into two scripts, `conda-setup.sh` and `conda-setup.bat`